WcaOnRails/app/views/committee_positions/new.html.erb

```diff
@@ -0,0 +1,8 @@
+<% provide(:title, 'Create a New Position In The ' + @committee.name) %>
+
+<div class="container new-team">
```

Comment:
`new-team` is not accurate here, moreover I see that you've removed `.new-team` class so it doesn't make any sense to put it here =)